py/qqa/run.py

```diff
@@ -294,5 +295,13 @@ def write_tables(indir, outdir):
 
     nightsfile = os.path.join(outdir, 'nights.html')
     webpages.tables.write_nights_table(nightsfile, exposures)
+    try:
+        os.mkdir(os.path.join(outdir, 'cal_files'))
+    except OSError:
+        i = 0
```

Comment:
It looks like this step is creating the cal_files output directory if needed.  Somewhat cleaner ways:
```python
os.makedirs(os.path.join(outdir, 'cal_files'), exist_ok=True)
```
or
```python
caldir = os.path.join(outdir, 'cal_files')
if not os.path.isdir(caldir):
    os.makedirs(caldir)
```
Also note that the python command "pass" can be used instead of "i=0" if you have other cases where you just need it to do nothing.